### PR TITLE
onboard to module catalog

### PR DESCRIPTION
**Pull request by ocofaigh**
### Description

onboard to module catalog

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json
Comment (SirSpidey) [inline ibm_catalog.json:65]:
```suggestion
                    "title": "Create Object Storage instance",
                    "description": "Create an IBM Cloud Object Storage instance."
                },
                {
                    "title": "Create Object Storage buckets",
                    "description": "Create both cross-regional and regional Object Storage buckets."
                },
                {
                    "title": "Create HMAC key",
                    "description": "Create an HMAC key for the Object Storage instance."
                },
                {
                    "title": "Attach access tags",
                    "description": "Attach access tags to the Object Storage instance."
                },
                {
                    "title": "Bucket retention",
                    "description": "Configure bucket retention."
                },
                {
                    "title": "Object versioning",
                    "description": "Supports enabling object versioning to keep multiple versions of an object in a bucket."
                },
                {
                    "title": "Archiving",
                    "description": "Add archive policies on the bucket."
                },
                {
                    "title": "Activity Tracking",
                    "description": "Integrate Activity Tracker events."
                },
                {
                    "title": "Monitoring",
                    "description": "Provision IBM Cloud Monitoring on Object Storage data."
                },
                {
                    "title": "KMS encryption",
                    "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
                },
                {
                    "title": "CBR",
                    "description": "Create context-based restriction (CBR) rules for both instance and buckets."
```

File: ibm_catalog.json
Comment (SirSpidey) [inline ibm_catalog.json:87]:
```suggestion
                    "label": "Complete (multiple Object Storage buckets with retention, encryption, tracking, and monitoring)",
```

File: ibm_catalog.json
Comment (SirSpidey) [inline ibm_catalog.json:104]:
```suggestion
                    "label": "IBM Cloud Framework for Financial Services compliant",
```

File: ibm_catalog.json
Comment (SirSpidey) [inline ibm_catalog.json:121]:
```suggestion
                    "label": "Object Storage bucket with a One Rate plan",
```

File: ibm_catalog.json
Comment (SirSpidey) [inline ibm_catalog.json:138]:
```suggestion
                    "label": "Object Storage bucket with replication",
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
see what you think

File: ibm_catalog.json
Comment (ocofaigh) [reply ibm_catalog.json:7]: @kbiegert can you confirm?

Comment (kbiegert) [reply ibm_catalog.json:7]: These should be the keywords that you can use on the search page, https://cloud.ibm.com/catalog

Comment (SirSpidey) [reply ibm_catalog.json:7]: @kbiegert what about tags?

File: ibm_catalog.json
Comment (kbiegert) [reply ibm_catalog.json:15]: Yes, as a search term

File: ibm_catalog.json
Comment (ocofaigh) [reply ibm_catalog.json]: moved to images directory

File: ibm_catalog.json
Comment (kbiegert) [reply ibm_catalog.json:7]: I believe tags determine the industry setting

**Review by SirSpidey — APPROVED** [review]
I don't want to hold this up. 

Separately, I think we need some more guidance about values: tags, keywords, product names, features... We might be able to rely on the  cloud docs catalog writers to provide that for both us and 3rd party users.

**Comment by SirSpidey** [comment]
@ocofaigh For features, you might be able to clean up here and now: 

**use the correct singular or plural in features.** 

For example, we have `Create Object Storage instance` (single) `Create Object Storage buckets` (plural). For these and all the features, it should be plural if you can provision more than one. I don't know the code well enough to get that correct.

**Comment by ocofaigh** [comment]
@SirSpidey The root level module supports optionally creating 1 instance and 1 bucket. The buckets submodule however supports creating multiple buckets. The problem is the module catalog does not expose submodules currently, so I think Ill update the feature to be single rather than plural for now

**Comment by ocofaigh** [comment]
Thanks @SirSpidey - can I get another approval off you here so I can merge?

**Review by SirSpidey — APPROVED** [review]
👍

**Pull request merged**
